src/classes/groundplaces.ts

```diff
@@ -27,23 +40,34 @@ export class GroundPlaces {
   /**
    * @description Create the stopGroup with the values given.
    * @param {StopClusterGpuid} stopClusterGpuid - Ground place unique identifier of the stopCluster parent.
-   * @param {StopGroup} stopGroupInfos - StopGroup informations.
-   * @returns {StopGroup}
+   * @param {StopInfos} stopGroupInfos - StopGroup informations.
+   * @returns {void|Error}
```

Comment:
It should returns the complete stopGroup AND adding it to the storage.

Reply:
This method now return the `StopGroup` created or an `Error` returned by the `applyGroundPlacesDiff()` method